packages/ai-jsx/src/core/prompts.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Basic prompts for LLMs.
+ * This file attemps to have suggestions to guide the thinking of inexperienced users
+ * to use good practices when using LLMs.
+ *
+ * Prompting is still a developing field. There are many different ways to prompt an LLM.
+ * This file attempts to provide some good defaults as a starting point.
+ */
+import * as LLMx from '../index.js';
+import { SystemMessage } from './completion';
+
+/**
+ * A (system) prompt collection with some useful defaults to get you started.
+ * By default all prompts are disabled to make it easier for combining them.
+ *
+ * We intend to improve this collection over time and welcome contributions.
+ * Potentially, the prompts can be adjusted based on the model being used.
```

Comment:
```suggestion
```

I tend not to want to write things like this in API docs. I think devs just want to know what the current state is.